vite.config.js

```diff
@@ -4,12 +4,12 @@ import vue from "@vitejs/plugin-vue2";
 import path from "node:path";
 import { readFileSync } from "node:fs";
 import copy from "rollup-plugin-copy";
-import pkg from './package.json';
-import * as child from 'child_process';
+import pkg from "./package.json";
```

Comment:
I will revert unwanted changes

Reply:
Aren't those changes caused by linter ?

Reply:
Yes, are linter changes. Should I just keep them? As the file is not changed due to this PR I wasn't sure if keep them.

Reply:
Please fix the sonar [issues](https://sonarcloud.io/project/issues?id=betaflight_betaflight-configurator&pullRequest=4351&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true)

Reply:
Yes safe to keep as you did not change them in the first place :)

Reply:
All the warnings are from the Three.js sample files brought from their library. I would prefer to avoid messing with it. Could it be that ok?

Reply:
Hmm sonar issues are only in CanvasRenderer.js (which is provided in node_modules examples) - so think we can leave it as is ?

Reply:
Agree.